### ISSUE TITLE
Footer / Header templates

### ISSUE THREAD

**Issue by finnhodgkin** [dwyl/dwyl-site#354]
Can I please write a short node script to add the footer / header to every html file in the redesign folder? We keep having to find/replace and some of the pages don't have Google analytics, etc. 

There are definitely higher priority issues, but if we're planning on keeping gh-pages for a while then it could save time overall.

**Comment by finnhodgkin** [comment]
Sorry dwylbot. Accidentally hit enter.

**Comment by markwilliamfirth** [comment]
Linked to https://github.com/dwyl/dwyl-site/issues/270

@finnhodgkin this was suggested by Cleo back in the day, but because we just wanted something up asap we didn't do it, but since we're blocked by #304 now it doesn't really matter anymore - do whatever saves us more time collectively 👍 

Although bear in mind that the site is eventually going to built in our stack https://github.com/dwyl/dwyl-site/issues/237#issuecomment-307075623

**Comment by markwilliamfirth** [comment]
Closing in favour of https://github.com/dwyl/dwyl-site/issues/270
